components/react/hooks/src/useCopyToClipboard/index.js

```diff
@@ -1,15 +1,12 @@
-import {useCallback} from 'react'
+import {useCallback, useState} from 'react'
 import copy from 'clipboard-copy'
 
 import useMountedState from '../useMountedState'
-import useSetState from '../useSetState'
 
 export default function useCopyToClipboard() {
   const isMounted = useMountedState()
-  const [state, setState] = useSetState({
-    value: undefined,
-    error: undefined
-  })
+  const [value, setValue] = useState()
```

Comment:
as you are setting these values always as one, what about using something like `const [{value, error}, setState]` so you can `setState({value, error})` afterwards without triggering 2 renders?

Reply:
done